src/RAJAPerfSuiteDriver.cpp

```diff
@@ -10,26 +10,44 @@
 
 #include <iostream>
 
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
+#include <mpi.h>
+#endif
+
 //------------------------------------------------------------------------------
 int main( int argc, char** argv )
 {
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
+  MPI_Init(&argc, &argv);
+#endif
+
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
+  int num_ranks;
+  MPI_Comm_size(MPI_COMM_WORLD, &num_ranks);
+  rajaperf::getCout() << "\n\nRunning with " << num_ranks << " MPI ranks..." << std::endl;
+#endif
+
   // STEP 1: Create suite executor object
   rajaperf::Executor executor(argc, argv);
 
   // STEP 2: Assemble kernels and variants to run
   executor.setupSuite();
 
-  // STEP 3: Report suite run summary 
+  // STEP 3: Report suite run summary
   //         (enable users to catch errors before entire suite is run)
-  executor.reportRunSummary(std::cout); 
+  executor.reportRunSummary(rajaperf::getCout());
```

Comment:
should we keep the ostream arg? it's a little more flexible if we need to do something else.